demos/python_demos/common/images_capture.py

```diff
@@ -0,0 +1,175 @@
+import os
+import sys
+import copy
+
+import cv2
+
+
+class InvalidInput(Exception):
+
+    def __init__(self, message):
+        self.message = message
+
+
+class OpenError(Exception):
+
+    def __init__(self, message):
+        self.message = message
+
+
+class ImagesCapture:
+
+    def __init__(self):
+        raise NotImplementedError
+
```

Comment:
```suggestion
```

`__init__` shouldn't raise an exception, because there's no reason to require subclasses to implement it (a subclass could have no `__init__` and still be a valid implementation). Just remove it.

Reply:
Done